app/assets/stylesheets/style/messages.scss

```diff
@@ -0,0 +1,62 @@
+.message {
+  border-top: 1px solid $light-gray-text;
+  margin-top: 0;
+  padding: 25px 0 20px;
+
+  &:first-child {
+    border-top: none;
```

Comment:
Does not appear to be working :cry:
You could use `border-bottom: ...` for all messages instead (you get a border for the last message too, but i think it looks ok)